src/main/java/com/google/sps/servlets/NotificationServlet.java

```diff
@@ -73,6 +73,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       query =  "SELECT follower_id FROM AnswerFollower WHERE answer_id = " + modifiedElementId;
       notificationUrl = "/question.html?id=" + modifiedElementId;
       notificationMessage = "Your answer was commented.";
+    } else if (typeOfNotification.equals("approval")) {
```

Comment:
It might be better to name this type of notification "requestApproval" or something more specific. I suspect you would eventually want to send a notification to the mentor who is signing up that says "You have been approved!", so it would be better to avoid having a generic notification of type "approval".

Reply:
Done!